src/App/index.js

```diff
@@ -16,12 +16,16 @@ const GlobalStyles = createGlobalStyle`
     font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   }
 
+  ul {
+    padding: 0;
+  }
+
   #app {
     @media only screen and (min-width: 400px) {
       min-height: 100vh;
       display: grid;
-      grid-template-columns: min-content;
-      justify-content: center;
+      // grid-template-columns: min-content;
```

Comment:
This is going to cause conflicts, need to discuss with @ibrahimbutt before merging.